src/core/frameworks/frameworks.ts

```diff
@@ -0,0 +1,385 @@
+      devServerCommand: "npm start",
+      devServerUrl: "http://localhost:8000",
+      outputLocation: "public"
+    }
+  },
+  {
+    id: "docusaurus",
+    name: "Docusaurus",
+    parent: "react",
+    packages: ["@docusaurus/core"],
+    config: {
+      // Same as base react
+    }
+  },
+  {
+    id: "react-static",
+    name: "React-static",
+    parent: "react",
+    packages: ["react-static"],
+    config: {
+      outputLocation: "dist"
+    }
+  },
+  {
+    id: "preact",
+    name: "Preact",
+    packages: ["preact"],
+    config: {
+      appBuildCommand: "npm run build",
+      devServerCommand: "npm run dev",
+      devServerUrl: "http://localhost:8080",
+      outputLocation: "build"
+    }
+  },
+  {
+    id: "vue",
+    name: "Vue.js",
+    packages: ["vue"],
+    config: {
+      appBuildCommand: "npm run build",
+      devServerCommand: "npm run serve",
+      devServerUrl: "http://localhost:8080",
+      outputLocation: "dist",
+    }
+  },
+  {
+    id: "vite",
+    name: "Vite",
+    parent: "vue",
+    packages: ["vite"],
+    config: {
+      devServerCommand: "npm run dev",
+      devServerUrl: "http://localhost:3000",
+    }
+  },
+  {
+    id: "nuxtjs",
+    name: "Nuxt.js",
+    parent: "vue",
+    files: ["nuxt.config.js"],
+    packages: ["nuxt"],
+    config: {
+      appBuildCommand: "npm run generate",
+      devServerCommand: "npm run dev",
+      devServerUrl: "http://localhost:3000",
+      outputLocation: "dist",
+    }
+  },
+  {
+    id: "vuepress",
+    name: "VuePress",
+    preempt: ["vue"],
+    packages: ["vuepress"],
+    config: {
+      appBuildCommand: "npm run build",
+      devServerCommand: "npm run dev",
+      devServerUrl: "http://localhost:8080",
+      outputLocation: "src/.vuepress/dist"
+    }
+  },
+  {
+    id: "svelte",
+    name: "Svelte",
+    packages: ["svelte"],
+    config: {
+      appBuildCommand: "npm run build",
+      devServerCommand: "npm run dev",
+      devServerUrl: "http://localhost:8080",
+      outputLocation: "public"
+    }
+  },
+  {
+    id: "riot",
+    name: "Riot.js",
+    packages: ["riot"],
+    config: {
+      appBuildCommand: "npm run build",
+      devServerCommand: "npm start",
+      devServerUrl: "http://localhost:3000",
+      outputLocation: "dist"
+    }
+  },
+  {
+    id: "stencil",
+    name: "Stencil.js",
+    files: ["stencil.config.ts"],
+    packages: ["@stencil/core"],
+    config: {
+      appBuildCommand: "npm run build",
+      devServerCommand: "npm start",
+      devServerUrl: "http://localhost:3333",
+      outputLocation: "www"
+    }
+  },
+  {
+    id: "aurelia",
+    name: "Aurelia",
+    packages: ["aurelia-bootstrapper", "aurelia-cli"],
+    config: {
+      appBuildCommand: "npm run build",
+      devServerCommand: "npm start",
+      devServerUrl: "http://localhost:8080",
+      outputLocation: "dist"
+    }
+  },
+  {
+    id: "ember",
+    name: "Ember.js",
+    packages: ["ember-cli"],
```

Comment:
Apps created by `ember new` all have `ember-cli` listed as a dependency, even older ones. I could not find any starters or example projects not having the cli listed as dependency, do you have one?

I'll add some other ember mandatory packages (like `ember-resolver`) as a safeguard, but I think it's preferable to avoid adding more complexity to the detection logic just for 1 framework, especially when we have other means using existing matchers.